platform/android/android.cmake

```diff
@@ -139,24 +139,11 @@ target_link_libraries(
         -Wl,--no-whole-archive
 )
 
-if(ANDROID_NATIVE_API_LEVEL VERSION_LESS 24)
-    target_sources(
-        mbgl-test-runner
-        PRIVATE ${PROJECT_SOURCE_DIR}/platform/android/src/test/http_file_source_test_stub.cpp
```

Comment:
Tbh I don't know well this part, but implementation in http_file_source_test_stub.cpp seem to do nothing, i.e. probably it is not even important if we have some implementation here or not (probably it should just compile).